recipes/pggb/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
     - time
     - vg 1.40.0
     - wfmash >=0.10.2
+    - python-igraph ==0.10.2
```

Comment:
Is this strict pin really necessary?

Reply:
Not really. I put it to avoid possible problems with future versions of igraph in case something changes in it. That is the version that we mainly tested. Would you suggest being relaxed and use '>='?

Reply:
You can check the changelog to see what changed. If you don't use experimental function (which are marked clearly in the docs), all behavioural changes should be the result of bugfixes, i.e. you really shouldn't be relying on the previous version's behaviour.

https://github.com/igraph/igraph/blob/master/CHANGELOG.md#0103---2022-12-30
https://github.com/igraph/python-igraph/blob/master/CHANGELOG.md#0103---2022-12-31

Latest version is 0.10.3, not 0.10.2.

Reply:
We don't use experimental functions, so we are good. I've updated the recipe's YAML accordingly, avoiding the constraint to a specific (old) version. Thank you both for the tips!